core/src/Text/Interpolation/Nyan/Core/Internal/Parser.hs

```diff
@@ -304,6 +330,11 @@ intPieceP = asum
 
   ]
   where
+    skipBlockComment' = asum
+      [ skipBlockComment "{-" "-}"
+      , try $ spaceChar *> skipBlockComment "{-" "-}"
+      ]
```

Comment:
I would rather write the clause with space removal as an extension of the "fast spacing" section above - to avoid extra rollbacks.

Generally, I find this approach good, you got the code grouped by the logic it is related to. But in our case performance matters (as we write a library), and the grammar is quite small, so we can afford some complication.

Reply:
>  But in our case performance matters

Yup, totally agree